bismuthcore/bismuthcore/legacy/peershandler.py

```diff
@@ -399,8 +401,8 @@ def peersync(self, subdata):
                             if pair not in peers_suggested and pair not in peers:
                                 peers_suggested[pair[0]] = pair[1]
                             peers[pair[0]] = pair[1]
-                        except:
-                            pass
+                        except Exception as e:
+                            self.app_log.debug(f"Connection to {pair[0]} {pair[1]} failed: {e}")
```

Comment:
The "pair" info is already logged with info level just above.
So in case of an exception, the short "not connectable" (typo in codebase) message is supposed to refer to the "connection" line just above. 
I'm not sure there's a need to duplicate the info.

Reply:
You're are correct. Will look into removing changes in `legacy/`